app/views/govuk_publishing_components/components/layout_for_public/_header_logo.html.erb

```diff
@@ -0,0 +1,41 @@
+<%
+  hide_logo_text ||= false
+  large_navbar ||= false
+  logo_text ||= t("components.layout_super_navigation_header.logo_text")
+  logotype_classes = %w(govuk-header__logotype gem-c-layout-super-navigation-header__logotype-crown)
+  logotype_classes << "gem-c-layout-super-navigation-header__logotype-crown--large-navbar" if large_navbar  
+%>
+
+<% unless hide_logo_text %>
+    <svg
```

Comment:
Nitpick: is something wrong with the indenting here? Looks like it's been double indented?

Reply:
Good spot, I've fixed this 👍